jaeger-mixin/dashboards.libsonnet

```diff
@@ -30,7 +30,7 @@ local g = (import 'grafana-builder/grafana.libsonnet') + {
 {
   grafanaDashboards+: {
     'jaeger-write.json':
-      g.dashboard('Jaeger / Write')
+      (g.dashboard('Jaeger / Write') + { uid: '5f26222aa7a3fb734f0cd4072cab43b3' })
```

Comment:
I've seen us using hashed uids in multiple places. Why? If they were human readable, they could be easy identifiers for dashboards, considering you might want to do something as `mixtool apply -t` someday

Reply:
The UIDs generated by grafana are random alpha-numeric IIRC; this was just a convenient way to get it to work, and is here to mirror what we do in prometheus-ksonnet.  We can also change this later.

Reply:
what @sh0rez is saying is that we can actually use human readable UIDs and push them to Grafana. Grafana only generates a random alphanum if it isn't provided with one. Providing a UID can give more human readable URLs too which is useful.